pdbx/containers.py

```diff
@@ -439,16 +457,18 @@ def append_attribute_extend_rows(self, attribute_name):
         attribute_name_lower = attribute_name.lower()
         if attribute_name_lower in self._catalog:
             index = self._attribute_name_list.index(
-                self._catalog[attribute_name_lower])
+                self._catalog[attribute_name_lower]
+            )
             self._attribute_name_list[index] = attribute_name
             self._catalog[attribute_name_lower] = attribute_name
             self.__lfh.write(
-                "Appending existing attribute %s\n" % attribute_name)
+                "Appending existing attribute %s\n" % attribute_name
+            )
         else:
             self._attribute_name_list.append(attribute_name)
             self._catalog[attribute_name_lower] = attribute_name
             # add a placeholder to any existing rows for the new attribute.
-            #if len(self._row_list) > 0:
+            # if len(self._row_list) > 0:
```

Comment:
remove this line?

Reply:
Remove the comment? The fix that black made was to add a space after the # sign. If it is safe to delete the commented code, I am OK with that but I don't have the history behind that comment or the technical knowledge to know if it should be a "TODO" in the future. I will defer to @sobolevnrm .

Reply:
Yes I would remove the comment. That condition is redundant, it's safe to loop over an empty list.

Reply:
Done.